index.html

```diff
@@ -50,6 +50,231 @@ <h3 class="box3">
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+          <p>
+              A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+          </p>  
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot3.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+           <p>
+                A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+           </p>
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="proj-myself">
+        <h1 class="my-title">
+            About Myself
+        </h1>
+        <p class="box2">
+            I'm a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+        <h3 class="box3">
+            LET'S CONNECT
+        </h3>
+        <ul class="box4">
+            <li class="social-media">
+                <img src="images/github.png" alt="github">
+            </li>
+            <li class="social-media">
+                <img src="images/Linkedin.png" alt="Linkedln">
+            </li> 
+            <li class="social-media">
+                <img src="images/fingers.png" alt="two fingers">
+            </li>
+            <li class="social-media">
+                <img src="images/twitter.png" alt="twitter">
+            </li>
+            
+            <li class="social-media">
+                <img src="images/m.png" alt="m">
+            </li>           
+        </ul>
+        <div class="div-button">
+            <button class="button">
+               Get my resume
+            </button>
```

Comment:
For each project, the button (e.g., "Get in touch") is a button tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below) as the previous reviewer mentioned. This makes your User Experience better.

<img width="284" alt="image" src="https://user-images.githubusercontent.com/77190203/148622820-cd349fc2-6516-436a-86c0-5fc982351365.png">

Reply:
Addressed 🙌